Sources/Sentry/SentryFileManager.m

```diff
@@ -429,20 +440,33 @@ - (NSDate *_Nullable)readTimestampLastInForeground
     return [NSDate sentry_fromIso8601String:timestampString];
 }
 
-- (NSString *)storeData:(NSData *)data toPath:(NSString *)path
+- (NSString *)storeData:(NSData *)data toUniqueJSONPath:(NSString *)path
 {
     @synchronized(self) {
-        NSString *finalPath = [path stringByAppendingPathComponent:[self uniqueAcendingJsonName]];
+        NSString *finalPath = [path stringByAppendingPathComponent:[self uniqueAscendingJsonName]];
         SENTRY_LOG_DEBUG(@"Writing to file: %@", finalPath);
-        [data writeToFile:finalPath options:NSDataWritingAtomic error:nil];
```

Comment:
We weren't checking the errors on some of these operations.

Reply:
Yes, we should check these. Thanks.